src/routes/SignIn.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { useFormik } from "formik";
+import {
+  Button,
+  Container,
+  TextField,
+  Typography,
+  Link,
+  Card,
+  Grid,
+} from "@material-ui/core";
+import { formStyles } from "../FormStyles";
+import { signInValidationSchema } from "../FormValidation";
+
+const SignIn = () => {
+  const formik = useFormik({
+    initialValues: {
+      email: "",
+      password: "",
+    },
+    validationSchema: signInValidationSchema,
+    onSubmit: (values) => {
+      // update next line to check login credentials against database
+      console.log(values);
+    },
+  });
```

Comment:
Something to consider that might be a pain later is having two different formik instances. This should be okay for now, we might be able to structure the app in a way that only has one formik instance, but multiple separate forms.

Reply:
by formik instances, do you mean separate invocations of `useFormik`? The latter has been standard in my experience

Reply:
- [let's discuss on Wednesday]